app/views/lookup_keys/_values.html.erb

```diff
@@ -17,10 +17,11 @@
   </thead>
 
   <tbody>
-  <%= f.fields_for :lookup_values do |lookup_values| %>
-    <%= render 'lookup_keys/value', :f => lookup_values, :is_param => is_param, :is_template => false %>
+  <%= f.fields_for :lookup_values do |lookup_value| %>
+    <%= render 'lookup_keys/value', :f => lookup_value, :is_param => is_param, :is_template => false, :hidden_value => f.object.hidden_value? %>
   <% end %>
   </tbody>
 </table>
-<%= new_child_fields_template(f, :lookup_values, { :partial => "lookup_keys/value", :form_builder_attrs => { :is_param => is_param, :is_template => true } }) %>
+<%= new_child_fields_template(f, :lookup_values, { :partial => "lookup_keys/value",
+                                                   :form_builder_attrs => { :is_param => is_param, :is_template => true, :hidden_value => f.object.hidden_value? } }) %>
```

Comment:
This is a new record and it doesn't always have a related lookup_key (even with the change in: https://github.com/theforeman/foreman/commit/6bbf50853340ac73ef37209aec8d0f3ef232ff70 this is still a problem with new lookup keys)